package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "badoo-styleguide",
-  "version": "0.0.82",
+  "version": "0.0.83",
```

Comment:
@langolf - I have a idea -  let's agree for next time that we upgrade version in separate commit, ideal flow will be:
1. we merge several features, for each feature in master after merge we update changelog
2. we rerun tests & update it
3. we update version in separate commit

Now no need to change, I will merge, but let's keep it in mind.